src/css/utilities/_avatar-size.scss

```diff
@@ -0,0 +1,12 @@
+$avatar_dimensions: (
+	"tiny": 4rem,
+	"small": 6rem,
+	"medium": 8rem,
+  "large": 12rem,
+  "hero": 14rem,
+);
+
+@mixin avatar-size($size: "tiny") {
+	height: map-get($avatar_dimensions, $size);
+	width: map-get($avatar_dimensions, $size);
+}
```

Comment:
We can move this into `c-avatar`, since it is component-specific logic. Our preferred style is an underscore prefix for component-specific variables and mixins, as well as dashes instead of underscores for word spaces. So, it would be `$_avatar-dimensions` and `_avatar-size`.

I know this seems pedantic, but we try to keep component-specific logic inside the component its utilized in to help both with maintenance and to communicate it shouldn't be used elsewhere improperly. If we find that this logic is needed elsewhere it signals the need for a refactor.

Reply:
Not pedantic at all, it's nice to have clarity in approach! That said, I'd like to chat about this a little more:

Is it expected to have mixins from one component's CSS being referenced in another component's? `avatar-size()` is referenced in several other Sass files, which was the reasoning behind peeling this piece of the CSS out into its own file: it felt more like a utility. It's the only part of the avatar component's CSS that isn't component-specific, but rather needs to be affected by its parent component.

I tend to interpret a mixin being in a component file to mean it's just for that file's use, which is how the rest of the codebase also seems to be set up; the only other non-third-party mixins I found in multiple files in the project were `hide()`, `link-states()`, `preserve-list-semantics()` and `heading-X()`, which aren't in component Sass files.  Maybe the avatar size mixin should be moved to the `src/css/logic/` folder? Or could be made more general as an image size mixin? Is there another aspect to this I'm missing?

Another option could be to just have the avatar component always be 100% height/width (perhaps with a max size in place as a fail-safe), and then it's up to the parent component to set the constraining value. The current sizes aren't providing a ton of consistancy as-is, so I'm not sure we'd lose much by eliminating the size map (it could just go back to being what was in the `_person.scss` file before). This might make things more self-contained while still having parents call the shots in sizing, which was the ultimate goal for all this. What do you think?

Reply:
The approach we're using is: 

Use an underscore for a mixin that's helpful to a single component. If more than one component could benefit from the mixin, promote it by creating a file for it in ['logic/`](https://github.com/a11yproject/a11yproject.com/tree/main/src/css/logic). So, based on your description, I think your call of moving it there is a good one.

As for the second option you propose, I think your initial call is a good one. I think this is more how to treat the mixin logic, and that's on me for not having good documentation. We have a little bit of it written down in [our styleguide](https://www.a11yproject.com/styleguide/), but it's something I'm slowly chipping away at when time permits.

Reply:
Great, thanks for helping me understand! I should be able to get some work in on this this weekend for this and the other changes.

Reply:
Summary of the changes I ended up making here:

* Renamed the mixin file to `_mixins.avatar-size.scss`
* Moved it into the `/logic` folder

I _think_ this is what we landed on per the above conversation, but wanted to check in on it since so much time has passed. Happy to revise as needed!